odoo/fields.py

```diff
@@ -3355,6 +3355,8 @@ def write(self, records, value):
                     property_definition.pop('value', None)
                 container[self.definition_record_field] = properties_definition
 
+                _logger.info('User #%i change definition of %r', records.env.user.id, container)
```

Comment:
"changed" ?

Also it does not really mention that it's related to the properties field from what I can see, you have to think that this message will appear mixed with a lot of unrelated others in the logs so it should maybe be a bit more verbose IMO.

Reply:
good catch
I added "Properties field: " at the beginning of the message (I didn't agree 100% with this log message, imo it's a bit useless to log a message each time we change the definition)